packages/react-core/src/components/Toolbar/examples/Toolbar.md

```diff
@@ -442,7 +505,15 @@ A toggle group can be used when you want to collapse a set of items into an over
 ```js
 import React from 'react';
 import { Toolbar, ToolbarItem, ToolbarContent, ToolbarToggleGroup, ToolbarGroup } from '@patternfly/react-core';
-import { Button, ButtonVariant, InputGroup, Select, SelectOption, SelectVariant, TextInput } from '@patternfly/react-core';
+import {
+  Button,
+  ButtonVariant,
+  InputGroup,
+  Select,
+  SelectOption,
+  SelectVariant,
+  TextInput
+} from '@patternfly/react-core';
```

Comment:
Imports still split here as well.

Reply:
unsplit imports for the added demo only, before I accidentally styled some imports and pushed the changes